#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_update_non_routables.dpatch by Yann Verley <yann.verley@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Update of non-routables with
## DP: http://www.iana.org/assignments/ipv4-address-space (last updated 2007-01-16).

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' fortified-1.1.0~/scripts/non-routables fortified-1.1.0/scripts/non-routables
--- fortified-1.1.0~/scripts/non-routables	2011-01-02 20:33:22.000000000 +0000
+++ fortified-1.1.0/scripts/non-routables	2011-01-02 20:35:06.000000000 +0000
@@ -1,87 +1,20 @@
 0.0.0.0/8
-1.0.0.0/8
-2.0.0.0/8
-5.0.0.0/8
-7.0.0.0/8
 10.0.0.0/8
-23.0.0.0/8
-27.0.0.0/8
-31.0.0.0/8
-36.0.0.0/8
-37.0.0.0/8
 39.0.0.0/8
-41.0.0.0/8
-42.0.0.0/8
-49.0.0.0/8
-50.0.0.0/8
-73.0.0.0/8
-74.0.0.0/8
-75.0.0.0/8
-76.0.0.0/8
-77.0.0.0/8
-78.0.0.0/8
-79.0.0.0/8
-89.0.0.0/8
-90.0.0.0/8
-91.0.0.0/8
-92.0.0.0/8
-93.0.0.0/8
-94.0.0.0/8
-95.0.0.0/8
-96.0.0.0/8
-97.0.0.0/8
-98.0.0.0/8
-99.0.0.0/8
-100.0.0.0/8
-101.0.0.0/8
 102.0.0.0/8
 103.0.0.0/8
 104.0.0.0/8
-105.0.0.0/8
 106.0.0.0/8
-107.0.0.0/8
-108.0.0.0/8
-109.0.0.0/8
-110.0.0.0/8
-111.0.0.0/8
-112.0.0.0/8
-113.0.0.0/8
-114.0.0.0/8
-115.0.0.0/8
-116.0.0.0/8
-117.0.0.0/8
-118.0.0.0/8
-119.0.0.0/8
-120.0.0.0/8
-121.0.0.0/8
-122.0.0.0/8
-123.0.0.0/8
-124.0.0.0/8
-125.0.0.0/8
-126.0.0.0/8
 127.0.0.0/8
 169.254.0.0/16
 172.16.0.0/12
-173.0.0.0/8
-174.0.0.0/8
-175.0.0.0/8
-176.0.0.0/8
-177.0.0.0/8
-178.0.0.0/8
 179.0.0.0/8
-180.0.0.0/8
-181.0.0.0/8
-182.0.0.0/8
-183.0.0.0/8
-184.0.0.0/8
 185.0.0.0/8
-186.0.0.0/8
-187.0.0.0/8
-189.0.0.0/8
-190.0.0.0/8
+192.0.0.0/24
 192.0.2.0/24
 192.168.0.0/16
-197.0.0.0/8
 198.18.0.0/15
-223.0.0.0/8
-224.0.0.0/3
+198.51.100.0/24
+203.0.113.0/24
+224.0.0.0/4
+240.0.0.0/4
